### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by newcomercurt**
This should be good to merge at this point. Page.js is working, nav bar is working, merge conflicts resolved.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (GeoffreyEmerson) [inline .editorconfig:9]: Is this file an atom config file? If you really need it in your working directory, add a line to .gitignore for it.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "Organization tool for casual gatherings.",
   "main": "src/app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/app.js"
```
Comment (GeoffreyEmerson) [inline package.json]: I seem to recall playing with this line to try to get Heroku working. Did you add this or is it a remnant of an older pull? Either way, it shouldn't be necessary. Nodemon should use "main" I think, and Heroku uses the Procfile.

File: .editorconfig (hunk 1 above)
Comment (newcomercurt) [inline .editorconfig:9]: It's not specific to atom it will make sure everyone's editor uses the same indenting and formatting style as we work on this project together.  

http://editorconfig.org/

File: package.json (hunk 2 above)
Comment (newcomercurt) [inline package.json]: I'll use nodemon, good call.

**Comment by newcomercurt** [comment]
@GeoffreyEmerson Fixes #54

**Comment by newcomercurt** [comment]
this addresses #9

**Comment by newcomercurt** [comment]
resolved merge conflicts

File: src/app.js [hunk 3]
```diff
@@ -5,14 +5,26 @@ var app = express();
...
+});
+
+app.get('/event*', function(req, res){
+  res.sendFile('/public/event.html',{root:__dirname + '/..'});
+});
+
+app.get('/admin*', function(req, res){
+  res.sendFile('/public/admin.html',{root:__dirname + '/..'});
+});
+
```
Comment (GeoffreyEmerson) [inline src/app.js]: Curt, could you revert the changes in this file? The removed line 11 covers everything you added.

**Comment by newcomercurt** [comment]
@GeoffreyEmerson initial commit for SPA work and page.js correction.

**Comment by newcomercurt** [comment]
@GeoffreyEmerson  serving the index file and hooked up logging for some of the routes we will have.
